maquetado/scss/detail.scss

```diff
@@ -0,0 +1,22 @@
+.title {
+  margin: 20px 0 0 0;
+}
+
+.field {
+  font-size: 19px;
+  font-weight: bold;
+}
+
+.field-description {
+  color: gray;
+  font-size: 19px;
+
+  &.strong {
+    font-weight: bold;
+  }
+}
+
+.fields-component {
```

Comment:
This class is not necessary.. you can delete it

Reply:
You mean I dont need those properties? I do that in order to use `justify-content: space-between;`. Or that I can put the properties with the style attribute in the html file?